client/disputes/info/index.tsx

```diff
@@ -42,15 +45,18 @@ const composeTransactionIdLink = ( dispute ) => {
 	);
 };
 
-const composeDisputeReason = ( dispute ) => {
+const composeDisputeReason = ( dispute: Dispute ): string => {
 	const reasonMapping = reasons[ dispute.reason ];
 	return reasonMapping
 		? reasonMapping.display
 		: formatStringValue( dispute.reason );
 };
 
-const Info = ( { dispute, isLoading } ) => {
-	const data = isLoading
+const Info = ( {
+	dispute,
+	isLoading,
+}: Pick< UseDisputeObject, 'dispute' | 'isLoading' > ): JSX.Element => {
```

Comment:
That's something new, what is this about (curious)?

Reply:
I've created `UseDisputeObject` as return value for `useDispute` but `Info` component received only 2 of 3 keys in that object. `Pick` allows to create a new type that picks only those 2 keys that we need.

Reply:
I see. I'd stick to the object as you did initially and update Info to receive all objects (even if it's not using them). That might be silly, but we want to go all the strict way forward.

Reply:
I'm not sure about this. 

AFAIU, `Pick` and `Omit` were created to handle these return objects that contain more than one return value. 

If we could just directly pass `UseDisputeObject` into `<Form/>` then it would be much cleaner.

However, the `useDispute` method is used in https://github.com/Automattic/woocommerce-payments/blob/develop/client/disputes/details/index.js#L24 and the return object **has to be** destructed as the code is using `dispute` and `isLoading` and `doAccept` separately. If we pass them all to the `<Form/>` component it's misleading as the `doAccept` is not used there.

Reply:
Understood,  let's keep this way for now, I'd like to revisit this once we have all changes in one PR.

Reply:
My opinion is similar to Vlad's. I think Pick/Omit should be avoided if it's a matter of just 2-3 extra properties that are not going to be used in a particular use case. Unless those extra props are not marked optional, in which case something like this would improve readability:

```typescript
export interface UseDisputeObject {
	dispute: Dispute;
	isLoading: boolean;
	doAccept: () => void;
};

// for use with <Form />
// pick a better name
interface UseDisputeForForm = Omit<UseDisputeObject, 'doAccept'>;
```

Reply:
Added one more comment in the thread above. Let's not use Omit/Pick - it's not consistent with our codebase and it'd be great to keep it as simple as possible.